src/test/java/bio/terra/MediaTypeUtilsTest.java

```diff
@@ -0,0 +1,75 @@
+package bio.terra;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import bio.terra.cloudfunctions.utils.MediaTypeUtils;
+import bio.terra.common.BaseTest;
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.util.logging.Logger;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.compressors.CompressorInputStream;
+import org.junit.Test;
+
+public class MediaTypeUtilsTest extends BaseTest {
+  private static final Logger logger = Logger.getLogger(MediaTypeUtilsTest.class.getName());
+
+  @Test
+  public void mockTGZTest() {
+    CompressorInputStream cis = null;
+    ArchiveInputStream ais = null;
+    BufferedInputStream bis = null;
+    try {
+      cis = MediaTypeUtils.createCompressorInputStream(MOCK_TGZ);
+      bis = new BufferedInputStream(cis);
+      ais = MediaTypeUtils.createArchiveInputStream(cis);
+      ArchiveEntry archiveEntry;
+      logger.info("mockTGZTest:");
```

Comment:
nit: are these logging statements necessary, here and in mockGZTest?

Reply:
Cleaned up comments.